straxen/contexts.py

```diff
@@ -74,6 +76,7 @@ def xenonnt_online(output_folder='./strax_data',
 
     st.storage = [straxen.RunDB(
         readonly=not we_are_the_daq,
+        minimum_run_number=_minimum_run_number,
```

Comment:
One question will this affect also data which is stored on disk? If I use this context and I ask for a run before 9271, the only protection the user has is the "forbid_creation_of"-setting is not it? Otherwise I would just make the new data without knowing it.

How about a user warning which pops up when starting the context. We can remove this warning with the next release again.

Reply:
Good question Daniel, it is jus the st.select_runs() where you scan the database that will be affected. It's not a problem at all to load old runs (if the lineage hash matches).